editoast/src/railjson/operation/delete.rs

```diff
@@ -0,0 +1,85 @@
+use super::{ObjectType, OperationError};
+use crate::response::ApiError;
+use diesel::{sql_query, PgConnection, RunQueryDsl};
+use rocket::serde::Deserialize;
+
+#[derive(Clone, Deserialize)]
+#[serde(crate = "rocket::serde")]
+pub struct DeleteOperation {
+    pub obj_type: ObjectType,
+    pub obj_id: String,
+}
+
+impl DeleteOperation {
+    pub fn apply(&self, infra_id: i32, conn: &PgConnection) -> Result<(), Box<dyn ApiError>> {
+        match sql_query(format!(
+            "DELETE FROM {} WHERE obj_id = '{}' AND infra_id = {}",
+            self.obj_type.get_table(),
+            self.obj_id,
```

Comment:
isn't the object ID user controlled? I'd feel better if this were properly escaped